external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -62,7 +62,7 @@ use crate::{
 use anyhow::{anyhow, Result};
 use crossbeam::channel::Sender;
 use im::ordmap::OrdMap;
-use lsp_server::{Request, RequestId};
```

Comment:
Gentle nudge that one day we should refactor this file into bite-sized chunks

Reply:
Agreed! How about this commitment - next time I want to make sizable changes to this file (other than emergency fixes perhaps), I will do the refactoring first?